plugins/wasm-go/Dockerfile

```diff
@@ -1,6 +1,8 @@
 ARG BUILDER=higress-registry.cn-hangzhou.cr.aliyuncs.com/plugins/wasm-go-builder:go1.19-tinygo0.27.0-oras1.0.0
 FROM $BUILDER as builder
 
+ENV GOPROXY https://goproxy.cn
```

Comment:
This may not be a good idea for developers outside the Chinese mainland.

Reply:
Check out this: https://github.com/alibaba/higress/blob/main/plugins/wasm-go/Makefile

Allowing executors to configure their own value might be better.

Reply:
> Check out this: https://github.com/alibaba/higress/blob/main/plugins/wasm-go/Makefile
> 
> Allowing executors to configure their own value might be better.

But for domestic developers, if you don’t add this option, it’s not very friendly, I didn’t think carefully, you’re right